postgresql-flexible-server/Migration/Powershell/SingleToFlexMigration.ps1

```diff
@@ -0,0 +1,60 @@
+# Script to trigger migration by directly invoking the end point through powershell script
+
+# Before running the script please run Connect-AzAccount and login with your azure credentials 
+
+# The URI parameter will need details of your flexible server's subscription and resource group along with the migration name you want to provide for this migration.
+$migrationName = "<migrationname>";
```

Comment:
Enclose parameters in param() which would prompt user to input the parameters for running this script